File: adhoc/mathiasvasconcelos/1961.c
#include<stdio.h>

int main() 
{
    
    int p,n,a,checker,i;
    
    while(scanf("%d %d",&p,&n)!=EOF)
    {
        int ar[n];
        checker = 0;
        for (i = 0; i < n;i++)
            scanf("%d",&ar[i]);
        for (i = 0;i < n-1;i++){
            a = ar[i+1]-ar[i];
            if (a<0)a *= -1;
            if (a>p){
                checker = 5;
                break;
            }
        }
        if (checker == 5)printf("GAME OVER\n");
        else printf("YOU WIN\n");
    }
    return 0;
}

File: adhoc/rafaelpetrucci/1961.py
case = list(map(int, input().split()))
pipes = list(map(int, input().split()))
count = 0
for i in range(case[1] - 1):
    if abs(pipes[i] - pipes[i + 1]) > case[0]:
        count += 1
        print('GAME OVER')
        break
if count == 0:
    print('YOU WIN')